metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -326,16 +326,107 @@ final class Doctor(
           _.element("thead")(
             _.element("tr")(
               _.element("th")(_.text("Build target"))
-                .element("th")(_.text("Scala"))
+                .element("th")(_.text("Type"))
                 .element("th")(_.text("Diagnostics"))
-                .element("th")(_.text("Goto definition"))
-                .element("th")(_.text("Completions"))
-                .element("th")(_.text("Find references"))
+                .element("th")(_.text("Interactive"))
+                .element("th")(_.text("Semanticdb"))
+                .element("th")(_.text("Debugging"))
+                .element("th")(_.text("Java support"))
                 .element("th")(_.text("Recommendation"))
             )
           ).element("tbody")(html => buildTargetRows(html, targetIds))
         )
     }
+
+    // Additional explanations
+    html.element("div")(
+      _.element("p")(
+        _.text(
+          "Diagnostics:"
+        )
+      ).element("ul")(
+        _.element("li")(
+          _.text(
+            s"${Icons.unicode.check} - connected build server showing diagnostics"
+          )
+        )
+          .element("li")(
+            _.text(s"${Icons.unicode.alert} - only syntactic errors")
+          )
+      )
+    )
+
+    html.element("div")(
+      _.element("p")(
+        _.text(
+          "Interactive features (completions, hover):"
+        )
+      ).element("ul")(
+        _.element("li")(
+          _.text(
+            s"${Icons.unicode.check} - supported Scala version"
+          )
+        )
+          .element("li")(
+            _.text(s"${Icons.unicode.error} - unsupported Scala version")
+          )
+      )
+    )
+
+    html.element("div")(
+      _.element("p")(
+        _.text(
+          "Semanticdb features (references, renames, go to implementation):"
+        )
+      ).element("ul")(
+        _.element("li")(
+          _.text(
+            s"${Icons.unicode.check} - build tool creates needed semanticdb files automatically"
+          )
+        )
+          .element("li")(
+            _.text(
+              s"${Icons.unicode.error} - no needed semanticdb indexes are produced"
```

Comment:
```suggestion
              s"${Icons.unicode.error} - semanticdb not being produced"
```

This is maybe another one that I'd just fully leave out unless there is an issue, wdyt?